src/bin/rdcore/rootmap.rs

```diff
@@ -53,6 +53,12 @@ pub fn rootmap(config: &RootMapConfig) -> Result<()> {
     // systemd-fstab-generator, and it defaults to read-only otherwise
     kargs.push("rw".into());
 
+    let rootflags = runcmd_output!("coreos-rootflags")?;
+    let rootflags = rootflags.trim();
```

Comment:
Nit: could be combined into previous line.

Reply:
We need to bind string return value before getting an `&str` out of it with `trim()`.